src/resolve/test-dns-query.c

```diff
@@ -0,0 +1,742 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "resolved-dns-query.h"
+#include "resolved-dns-rr.h"
+#include "resolved-dns-scope.h"
+#include "resolved-dns-server.h"
+#include "resolved-link.h"
+#include "resolved-manager.h"
+
+#include "log.h"
```

Comment:
same as the other PR, these are internal headers so group them with the above and sort alphabetically